benchmark/es/string-repeat.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const assert = require('assert');
+const common = require('../common.js');
+
+const configs = {
+  n: [1e3],
+  mode: ['Array', 'repeat'],
+  encoding: ['ascii', 'utf8'],
+  size: [1e1, 1e3, 1e6],
+};
+
+const bench = common.createBenchmark(main, configs);
+
+function main(conf) {
+  const n = +conf.n;
+  const size = +conf.size;
+  const character = conf.encoding === 'ascii' ? 'a' : '\ud83d\udc0e'; // '🐎'
+
+  let str;
+
+  if (conf.mode === 'Array') {
+    bench.start();
+    for (let i = 0; i < n; i++) str = new Array(size + 1).join(character);
```

Comment:
I like this style but I think it's not widely used in the codebase.
Is it better to put the `for` body in the next line?

Also is it ok to use `let` in for loops like this now?